### PR TITLE
Improve the CLI

### PR DESCRIPTION
**Pull request by timrogers**
This PR introduces a number of improvements to the command line tools included with the package.

Specifically, it:

* adds a new `getNameForDevice` which returns a device's name (e.g. "Logitech Litra Glow")
* updates all of the CLIs so you can specify a `--serial-number`/`-s` to target a specific device when you have multiple devices connected
* adds a new `litra-identify` CLI for getting the serial numbers of your Litra devices when multiple are connected
* switches to using the `commander` package for a better CLI experience and more robust argument parsing

### CODE REVIEW

**Comment by timrogers** [comment]
@leggetter Would you be able to give the new `--serial-number` option and `litra-identify` commands a go, since you have multiple devices?

**Comment by leggetter** [comment]
> @leggetter Would you be able to give the new `--serial-number` option and `litra-identify` commands a go, since you have multiple devices?

tl;dr - the `-s` works to control individual devices.

I didn't globally install as I have a clash with my own commands called `litra-on` and `litra-off`. So, I'm running the commands directly using `node dist/path/to/command.js`.

```
timrogers/cli-improvements $ node dist/commonjs/cli/litra-identify.js
Found 2 connected Litra devices
Flashing light on Logitech Litra Beam device with serial number SERIAL_ONE...
Press Enter to continueFlashing light on Logitech Litra Beam device with serial number SERIAL_TWO...

Press Enter to continue%                                                 
timrogers/cli-improvements $ node dist/commonjs/cli/litra-on.js -s SERIAL_ONE
timrogers/cli-improvements $ node dist/commonjs/cli/litra-on.js -s SERIAL_TWO
```

The flashing of the light is a great idea to identify them, and the console output seems to suggest this should repeat until enter is pressed; however, the script seems to continue and exist without user intervention.

My next comment is the `...` after the serial number makes it slightly harder to copy the value by double-clicking onit with the mouse because that means the `...` is also highlighted rather than just the serial number.

Maybe that's just because of how I'm running the commands?
